package.json

```diff
@@ -19,6 +19,7 @@
     "ast-types": "0.9.4",
     "babel-code-frame": "6.22.0",
     "babylon": "6.15.0",
+    "colors": "1.1.2",
```

Comment:
Oh, I'm not experienced in cli-tools so it hour it's better to get new dep :) I can rewrite without it. Just need some time for research.